### PR TITLE
Login_Page

### PR DESCRIPTION
**Pull request by Apoorv-cloud**

### CODE REVIEW

File: lib/main.dart [hunk 1]
```diff
@@ -27,39 +41,261 @@ class MyHomePage extends StatefulWidget {
...
+          width: 30,
+        ),)
+        ]
+
+    ),
+
+      backgroundColor: Colors.white,
+      //backgroundColor: Colors.white,
+
+   body: Stack(
```
Comment (siddhantverma95) [inline lib/main.dart:73]: Can we use SafeArea widget here. In ios and some other android devices with notch the bottom is overlapped with home button.

File: lib/main.dart [hunk 2]
```diff
@@ -27,39 +41,261 @@ class MyHomePage extends StatefulWidget {
...
+            right: 0,
+            left: 0,
+            child: Column(children: [
+              Text('Don\'t have an account? Sign Up!',
+                style: TextStyle(
+                  decoration: TextDecoration.underline,
+                  color: Colors.blue[500],
+                  fontSize: 16,
+                ),)
+            ],),)
```
Comment (siddhantverma95) [inline lib/main.dart:229]: Right now in this current situation if I want to make this whole view scrollable, I can't do that cause positioned uses particular points on screen and in scroll view there are no particular point. Plus using positioned everywhere is not a good idea for flexible design. Try to make it more flexible.. TIPS: try to use expanded and spacer widgets to achieve that.

File: lib/main.dart [hunk 3]
```diff
@@ -27,39 +41,261 @@ class MyHomePage extends StatefulWidget {
...
+              child: Image.asset(
+                img,
+                height: 20,
+              ),
             ),
-          ],
+          ),
+        ),
+      ),
+    ));
```
Comment (siddhantverma95) [inline lib/main.dart:264]: We can use commas after every bracket to shift it automatically to the new line each. This makes code more readable for other person. Also try to use a formatter plugin that formats your code after every save. For VScode you can use Prettier plugin.

**Review by siddhantverma95 — CHANGES_REQUESTED** [review]

**Comment by Apoorv-cloud** [comment]
All changes done Sir

**Pull request closed without merging**